rust/osm2lanes/src/transform/tags_to_lanes/modes/bus.rs

```diff
@@ -225,7 +276,7 @@ fn bus_lanes(
             }
             if let Some(backward) = backward {
                 let backward_access = Access::split(backward).map_err(|a| {
-                    RoadError::from(TagsToLanesMsg::unsupported(
+                    RoadError::Msg(TagsToLanesMsg::unsupported(
```

Comment:
do you remember why you needed to change this? it works as well, I just don't know which is better :P

Reply:
Something silly from my IDE while I was merging. Turns out that wasn't what was breaking the build. We can probably roll this back and I can turn off that warning somehow.
![from error](https://user-images.githubusercontent.com/3784591/164878618-a70b58e3-bcd3-4e81-8eee-ff2ab0763c5e.png)

Reply:
it's fine, I was just curious